lib/livebook_web/components/core_components.ex

```diff
@@ -118,13 +124,13 @@ defmodule LivebookWeb.CoreComponents do
       @kind == :error && "border-red-500"
     ]}>
       <div
-        :if={@message}
+        :if={@message && @inner_block == []}
```

Comment:
Do we need this change, given that we already check that both cannot be given at the same time?

Reply:
```suggestion
        :if={@message}
```